docs-gen/content/docs/getting-started/installation.md

```diff
@@ -39,6 +39,8 @@ Valid configuration options to override the [Default `values.yaml`]({{< ghblob "
 | `manager.argo.containerDefaults`                          | Container Spec defaults to be used for Argo workflow pods created by the operator                                                                                                                                   |
 | `manager.argo.metadata`                                   | Container Metadata defaults to be used for Argo workflow pods created by the operator                                                                                                                               |
 | `manager.argo.ttlStrategy`                                | [TTL Strategy](https://argoproj.github.io/argo-workflows/fields/#ttlstrategy) used for all Argo Workflows                                                                                                           |
+| `manager.argo.stepTimeoutSeconds.default`                 | The maximum number of seconds that each non-compiler workflow step can run before being terminated - defaults to 300 (5m)                                                                                           |
+| `manager.argo.stepTimeoutSeconds.compile`                 | The maximum number of seconds that each compiler workflow step can run before being terminated - defaults to 1800 (60m)                                                                                             |
```

Comment:
...especially the 1800s being 30m, not 60m 😉
```suggestion
| `manager.argo.stepTimeoutSeconds.compile`                 | Timeout in seconds for compiler steps - defaults to 1800 (30m)                                                                                                                                                      |
| `manager.argo.stepTimeoutSeconds.default`                 | Default [timeout in seconds](https://argoproj.github.io/argo-workflows/walk-through/timeouts/) for workflow steps - defaults to 300 (5m)                                                                            |

```

Reply:
🤦